components/tools/OmeroPy/test/integration/test_bigImages.py

```diff
@@ -111,10 +112,12 @@ def testRenderRegion(self):
 
         # get the cropped image
         cropped = img_array[y:y2, x:x2, :]      # ... so we can crop to region
-        cropped_img = Image.fromarray(cropped)
+        buf = StringIO()
```

Comment:
did not notice any problem, will close it for now since ``pillow`` doesn't work and run test differently